webpack/assets/javascripts/react_app/components/common/Loader.js

```diff
@@ -3,12 +3,13 @@ import {STATUS} from '../../constants';
 import MessageBox from './MessageBox';
 import './LoaderStyles.css';
 
-const Loader = ({ status, children }) => {
+const Loader = ({ status, children, spinnerSize }) => {
 let content;
 
+spinnerSize = spinnerSize || 'lg';
```

Comment:
This should be done using a default parameter value in the function definition - 
```js
const Loader = ({ status, children, spinnerSize = 'lg' }) => {
...
```